README.md

```diff
@@ -140,6 +140,12 @@ OpenDAL is an active open-source project. We are always open to people who want
 
 - [QuestDB](https://github.com/questdb/questdb): An open-source time-series database for high throughput ingestion and fast SQL queries with operational simplicity.
 
+## Branding
+
+The first and most prominent mentions must use the full form: **Apache OpenDAL®** of the name for any individual usage (webpage, handout, slides, etc.) Depending on the context and writing style, you should use the full form of the name sufficiently often to ensure that readers clearly understand the association of both the OpenDAL project and the OpenDAL software product to the ASF as the parent organization.
```

Comment:
Is "Apache OpenDAL" a registered trademark? If no, perhaps `™` is formal to use.

Reply:
You may ask `trademark@apache.org` if uncertain.

Reply:
> Is "Apache OpenDAL" a registered trademark? If no, perhaps `™` is formal to use.

I'm following https://www.apache.org/foundation/marks/guide:

> The first and most prominent mentions must use the full form: "Apache Hadoop®" of the name for any individual usage (webpage, handout, slides, etc.) Depending on the context and writing style, you should use the full form of the name sufficiently often to ensure that readers clearly understand the association of both the Hadoop project and the Hadoop software product to the ASF as the parent organization.

Reply:
I have checked https://whimsy.apache.org/brand/list, and `Apache OpenDAL` is not registered.